### ISSUE TITLE
[BUG] This just stopped working

### ISSUE THREAD

**Issue by polykatt** [Darkhax-Minecraft/Tips#144]
### Minecraft Version

1.20.1

### Mod Version

12.1.7

### Mod Loader

Fabric

### What environment are you running the mod in?

Client

### Issue Description

I think it might just be because of this latest update because it worked fine before

**Comment by Darkhax** [comment]
Hello, thanks for reporting this issue. The 12.1.7 update changed a lot of things so it's definitely possible something broke. Could you please explain what you mean when you say it's not working? Like is it crashing or are the tips just not being displayed? I would also appreciate it if you could upload your log file and the config file here so I can check those for more information.

**Comment by polykatt** [comment]
Yeah, so I noticed that tips are not showing on the world creation screen. But they are there on the pause menu and when logging out of a world.

[latest.log](https://github.com/user-attachments/files/16234509/latest.log)

I haven't changed anything in the config file

[tips.json](https://github.com/user-attachments/files/16234540/tips.json)

**Comment by Darkhax** [comment]
Thank you for verifying this issue. I've also been able to verify it and should have a fix out soon.

**Comment by Darkhax** [comment]
This issue should be fixed in 12.1.8, please try it out and let me know how it goes. The file was just uploaded to [CurseForge](https://www.curseforge.com/minecraft/mc-mods/tips/files/5547512) and [Modrinth](https://modrinth.com/mod/tips/version/uq5F0GNj) so it may take a few minutes to show up.

**Comment by polykatt** [comment]
Yeah so the tips on world creation screen don't cycle. It displays the same one about wandering traders. And on the pause menu and logging out they don't seem to be cycling randomly - they start at the same one about wandering traders then follow the same sequence of tips every time.
[latest.log](https://github.com/user-attachments/files/16322898/latest.log)

Edit: Actually it seems like its not just starting at the same one tip about wandering traders but it does seem to start at some random tip and then follow the same sequence instead of random cycling. This seems to change every time I relaunch the game. Also they do seem to be cycling on the world creation screen now. I think maybe things are mostly fine now. Not so sure but I guess its okay - maybe with more playtime I'll can see what issues are really sticking.

**Comment by Darkhax** [comment]
@polykatt Thanks for letting me know. I will take a look at the tip selection logic and see if there's anything wrong.
